atomic_reactor/plugins/post_import_image.py

```diff
@@ -77,9 +77,9 @@ def run(self):
                                   build_json_dir=self.build_json_dir,
                                   namespace=metadata.get('namespace', None))
         osbs = OSBS(osbs_conf, osbs_conf)
-
+        i = None
```

Comment:
It would be better to rename this variable to smth like 'imagestream', as its complicated to find all instances of 'i' in the code